FlowCal/mef.py

```diff
@@ -752,6 +801,79 @@ def get_transform_fxn(data_beads,
     taken at the same settings as the calibration beads, for all channels
     in `mef_channels`.
 
+    Examples
+    --------
+    Here is a simple application of this function:
+
+    >>> transform_fxn = FlowCal.mef.get_transform_fxn(
+    ...     beads_data,
+    ...     mef_channels=['FL1', 'FL3'],
+    ...     mef_values=[np.array([    0,   646,   1704,   4827,
+    ...                           15991, 47609, 135896, 273006],
+    ...                 np.array([    0,  1614,   4035,   12025,
+    ...                           31896, 95682, 353225, 1077421]],
+    ...     )
+    >>> sample_mef = transform_fxn(sample_rfi, ['FL1', 'FL3'])
```

Comment:
I prefer including the parameter names to be as specific as possible, e.g.

`>>> sample_mef = transform_fxn(data_rfi=sample_rfi, channels=['FL1', 'FL3'])`

Up to you, though.